src/app/landing/landing.component.ts

```diff
@@ -0,0 +1,31 @@
+import { Component, ChangeDetectionStrategy, OnInit } from '@angular/core';
+import { Router } from '@angular/router';
+import { AuthService } from 'src/app/auth/auth.service';
+import { filter } from 'rxjs/operators';
+
+@Component({
+  selector: 'cb-landing',
+  templateUrl: './landing.component.html',
+  styleUrls: ['./landing.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class LandingComponent implements OnInit {
+  constructor(
+    private authService: AuthService,
+    private router: Router,
+  ) {}
+
+  public ngOnInit(): void {
+    this.authService.isAuthenticated()
+      .pipe(filter(isAuthonticated => !!isAuthonticated))
+      .subscribe(() => this.router.navigate(['/recipes']));
```

Comment:
No `unsubscribe` from this observable

Reply:
Yeah 👍🏾, I've already created an issue for this
https://github.com/dbl7/cookbook/issues/8